lineman/app/components/thread_page/proposal_collapsed/proposal_collapsed.haml

```diff
@@ -1,9 +1,11 @@
 %a.proposal-collapsed.media{href: '', id: 'proposal-{{proposal.key}}'}
   .sr-only
     {{proposal.name}}.
-    %closing_in{time: 'proposal.closingAt'}.
+    %closing_in{ng-if: 'proposal.isActive()', time: 'proposal.closingAt' }.
```

Comment:
Agree! `proposal_closing_time{proposal: 'proposal'}` maybe? and remove all the closing_ins?